src/test/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/PersonControllerTest.java

```diff
@@ -31,7 +31,7 @@
     @MockBean
     private PersonService personService;
 
-    private ObjectMapper objectMapper = new ObjectMapper();
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Prefer @Autowired for controllers but for now, it's ok as Person class doesn't have complicated objects to jsonize